code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -390,6 +390,11 @@
 	var/image/warcry = image('icons/mob/talk.dmi', user, icon_state = "warcry")
 	user.add_emote_overlay(warcry)
 
+/datum/emote/living/carbon/human/warcry/spartan
+	key = "finishthefight"
+	key_third_person = "finishesthefight"
+	message = "is finishing the fight."
```

Comment:
???

Reply:
[Tivi doesn't know.](https://youtu.be/jiN84inW1Tg?si=xk9PDd9ikKq6C2y7)

Reply:
People can do this better with the current existing kbs, just remove it